### ISSUE TITLE
Support : Need sample data to create a media file

### ISSUE THREAD

**Issue by Vanitha-Ramkumar** [akeneo/pim-api-docs#76]
Hi,

I struck with creating the media file. If you share sample data to create a media file it will be more helpful.

Thanks in advance.

**Comment by stbCreo** [comment]
Hi. Although there is no answer to this, I give it a chance ;)
How could a POST look like, that retrieves an image from an URL and appends it to a product?

I tried this on ubuntu with Akeneo 2.3:
```
curl -v -X POST http://myPIM.com/api/rest/v1/media-files \
    -H "Content-type: multipart/form-data" \
    -H "Authorization: Bearer token" \
    -f "product_model={"code":"0123456", "attribute":"product_picture_main", "scope": "null", "locale":"null"}
    -f "file='http://mypictureURL.com/800/1111555577.jpg'"
```

Giving the error: "The requested URL returned error: 422 Unprocessable Entity"
Code and attribute are correct. Attribute is also neither scopable or localizable.
Variations of quotation marks and apostrophe did not work, but probably I did not find the right combination.

Does anyone have an idea on how to do this?
(I also posted this in the slack channel, but maybe it is more visible here)
